addons/html_editor/static/src/editor.js

```diff
@@ -47,27 +52,21 @@ function sortPlugins(plugins) {
     return result;
 }
 
-/**
- * @typedef {typeof import("./plugin").Plugin} PluginConstructor
- */
-
 export class Editor {
     /**
-     * @param { PluginConstructor[] } Plugins
      * @param { EditorConfig } config
      * @param {*} services
      */
     constructor(config, services) {
         this.isDestroyed = false;
-        /** @type { EditorConfig } **/
         this.config = config;
         this.services = services;
         this.plugins = [];
         /** @type { HTMLElement } **/
         this.editable = null;
         /** @type { Document } **/
         this.document = null;
-        /** @type { SharedMethods } **/
+        /** @ts-ignore  @type { SharedMethods } **/
```

Comment:
ts-ignore ?

Reply:
yes, because my typechecker sees that `{}` is not a valid SharedMethod type.